src/Native/Runtime/MathHelpers.cpp

```diff
@@ -82,6 +82,60 @@ EXTERN_C REDHAWK_API UInt64 REDHAWK_CALLCONV RhpDbl2ULng(double val)
     return((UInt64)val);
 }
 
+// CORERT Specific - on Project N the arguments to these helpers are inverted
+#ifdef CORERT
+#include <cmath>
+
+EXTERN_C REDHAWK_API float REDHAWK_CALLCONV RhpFltRem(float dividend, float divisor)
+{
+    //
+    // From the ECMA standard:
+    //
+    // If [divisor] is zero or [dividend] is infinity
+    //   the result is NaN.
+    // If [divisor] is infinity,
+    //   the result is [dividend] (negated for -infinity***).
+    //
+    // ***"negated for -infinity" has been removed from the spec
+    //
+
+    if (divisor==0 || !isfinite(dividend))
```

Comment:
Fixed. I didn't really want to use the `std` version for the other one either, but the non-std version was [failing to build on Linux](http://stackoverflow.com/questions/33770374/why-is-isnan-ambiguous-and-how-to-avoid-it).